src/TextareaAutosize.js

```diff
@@ -87,6 +93,13 @@ export default class TextareaAutosize extends React.Component {
     // Re-render with the new content then recalculate the height as required.
     this.onNextFrameActionId = onNextFrame(this._resizeComponent);
   }
+  
+  componentWillUpdate(nextProps, nextState) {
```

Comment:
Also what if parent decides to re-render (calls its `setState` from within the callback)?